This patch sets the default value of the number of "workers" in the
Swift configuration files for a single CPU system. The default based on
the number of CPUs (cores) found may not be efficient on a system with
a large number of cores. This patch is not suitable for the upstream.

--- swift-2.3.0/etc/account-server.conf-sample.~2~	2016-02-08 00:02:16.969218591 -0800
+++ swift-2.3.0/etc/account-server.conf-sample	2016-02-08 00:02:17.037019168 -0800
@@ -10,8 +10,9 @@ bind_port = 6002
 # disable_fallocate = false
 #
 # Use an integer to override the number of pre-forked processes that will
-# accept connections.
-# workers = auto
+# accept connections.  Use "auto" or default for the number of effective cpu
+# cores in the system.
+workers = 1
 #
 # Maximum concurrent requests per worker
 # max_clients = 1024
--- swift-2.3.0/etc/container-server.conf-sample.~2~	2016-02-08 00:02:16.969741046 -0800
+++ swift-2.3.0/etc/container-server.conf-sample	2016-02-08 00:02:17.037565309 -0800
@@ -10,8 +10,9 @@ bind_port = 6001
 # disable_fallocate = false
 #
 # Use an integer to override the number of pre-forked processes that will
-# accept connections.
-# workers = auto
+# accept connections.  Use "auto" or default for the number of effective cpu
+# cores in the system.
+workers = 1
 #
 # Maximum concurrent requests per worker
 # max_clients = 1024
--- swift-2.3.0/etc/object-server.conf-sample.~2~	2016-02-08 00:02:16.971225158 -0800
+++ swift-2.3.0/etc/object-server.conf-sample	2016-02-08 00:02:17.038057162 -0800
@@ -12,8 +12,9 @@ bind_port = 6000
 # expiring_objects_account_name = expiring_objects
 #
 # Use an integer to override the number of pre-forked processes that will
-# accept connections.
-# workers = auto
+# accept connections.  Use "auto" or default for the number of effective cpu
+# cores in the system.
+workers = 1
 #
 # Maximum concurrent requests per worker
 # max_clients = 1024
--- swift-2.3.0/etc/proxy-server.conf-sample.~1~	2015-04-30 06:57:42.000000000 -0700
+++ swift-2.3.0/etc/proxy-server.conf-sample	2016-02-08 00:02:17.038685067 -0800
@@ -23,10 +23,10 @@ bind_port = 8080
 # disallowed_sections = swift.valid_api_versions, container_quotas, tempurl
 
 # Use an integer to override the number of pre-forked processes that will
-# accept connections.  Should default to the number of effective cpu
+# accept connections.  Use "auto" or default for the number of effective cpu
 # cores in the system.  It's worth noting that individual workers will
 # use many eventlet co-routines to service multiple concurrent requests.
-# workers = auto
+workers = 1
 #
 # Maximum concurrent requests per worker
 # max_clients = 1024
